tests/Browser/BookingsSearchTest.php

```diff
@@ -45,4 +46,71 @@ public function testUserCanViewRoomsAvailabilitiesCalender()
+
+            $browser->waitUntilVue('calendarRooms[0].name', $room->name, '@calendar-view-table');
+            $browser->waitUntilVue('calendarRooms[0].availabilities[0].id', $a1->id, '@calendar-view-table');
+
+            // verify calendar navigation
+            $leftDelimiter = 8;
+            $rightDelimiter = 21;
+            $browser->assertVue('leftHourDelimiter', $leftDelimiter, '@calendar-view-table');
+            $browser->assertVue('rightHourDelimiter', $rightDelimiter, '@calendar-view-table');
+            $browser->click('@hours-left');
+            $browser->waitUntilVue('leftHourDelimiter', $leftDelimiter - 1,'@calendar-view-table');
+            $browser->click('@hours-left');
+            $browser->waitUntilVue('leftHourDelimiter', $leftDelimiter - 2,'@calendar-view-table');
+            $browser->assertVue('rightHourDelimiter', $rightDelimiter - 2,'@calendar-view-table');
+
+            // verify that the loaded content is for today
+            $browser->assertVue('dateSelected', today()->toDateString(), '@calendar-view-table');
+            $browser->assertVue('calendarRooms[0].name', $room->name, '@calendar-view-table');
+            $browser->assertVue('calendarRooms[0].day_breakdown[7][0].closed', false, '@calendar-view-table');
+            $browser->assertVue('calendarRooms[0].day_breakdown[6][0].closed', true, '@calendar-view-table');
+            $browser->assertVue('calendarRooms[0].day_breakdown[8][0].closed', true, '@calendar-view-table');
+
+            // setup different availabilities for tomorrow
+            $tomorrow = today()->addDay();
+            $a2 = Availability::create([
+                'weekday' => $tomorrow->englishDayOfWeek,
+                'opening_hours' => '09:00',
+                'closing_hours' => '10:00',
+                'room_id' => $room->id
+            ]);
+
+            $browser->within(new DateTimePicker('date_selected'), function($browser) use ($tomorrow) {
+                $browser->selectDate($tomorrow);
+            });
+
+            $browser->waitUntilVue('dateSelected', $tomorrow->toDateString(), '@calendar-view-table');
+            $browser->waitUntilVue('calendarRooms[0].availabilities[0].id', $a2->id, '@calendar-view-table');
+            // date should change to reflect tomorrow's availabilities
+            $browser->assertVue('calendarRooms[0].day_breakdown[7][0].closed', true, '@calendar-view-table');
+            $browser->assertVue('calendarRooms[0].day_breakdown[8][0].closed', true, '@calendar-view-table');
+            $browser->assertVue('calendarRooms[0].day_breakdown[7][0].closed', true, '@calendar-view-table');
+            $browser->assertVue('calendarRooms[0].day_breakdown[8][0].closed', true, '@calendar-view-table');
+            $browser->assertVue('calendarRooms[0].day_breakdown[9][0].closed', false, '@calendar-view-table');
+            $browser->assertVue('calendarRooms[0].day_breakdown[10][0].closed', true, '@calendar-view-table');
```

Comment:
If this can't be encapsulated into a component with a meaningful method name, plz add some comments instead to explain whats actually being asserted here.

Reply:
Same for the other section of long selectors in this file

Reply:
because asserts are not meant to be inside components? But I'll add comment ez pz

Reply:
@alexstojda clarified existing comments, added more comments, commented the comments 🤸